java-tools/OsmAndServer/src/main/java/net/osmand/server/controllers/pub/RoutingController.java

```diff
@@ -161,11 +177,15 @@ public RoutingParameter(String key,  String name, String description, String gro
 	@RequestMapping(path = "/routing-modes", produces = {MediaType.APPLICATION_JSON_VALUE})
 	public ResponseEntity<?> routingParams() {
 		Map<String, RoutingMode> routers = new LinkedHashMap<>();
-		RoutingParameter nativeRouting = new RoutingParameter("nativerouting", "Development", 
-				"[Dev] C++ routing", true);
+		RoutingParameter hhRouting = new RoutingParameter("hhrouting", "Development",
+				"[Dev] Use HH (auto)", true);
+		RoutingParameter hhOnly = new RoutingParameter("hhonly", "Development",
```

Comment:
Not needed for UI (only for API)

Reply:
>Not needed for UI (only for API)

Which one isn't needed for the UI?

Reply:
Comment to last line ```RoutingParameter hhOnly = new RoutingParameter("hhonly", "Development",```

Reply:
9f8dc4f0c565a20b9232ab11e89018c910865e69